network/dime_message.ksy

```diff
@@ -0,0 +1,76 @@
+meta:
+  id: dime_message
+  title: DIME (Direct Internet Message Encapsulation) Message
+  xref:
+    wikidata: Q1227457
+  license: CC0-1.0
+  bit-endian: be
+  endian: be
+doc: |
+  Direct Internet Message Encapsulation (DIME)
+  is an old Microsoft specification for sending and receiving
+  SOAP messages along with additional attachments,
+  like binary files, XML fragments, and even other
+  SOAP messages, using standard transport protocols like HTTP.
+doc-ref: 
+  - http://xml.coverpages.org/draft-nielsen-dime-02.txt
+  - https://docs.microsoft.com/en-us/archive/msdn-magazine/2002/december/sending-files-attachments-and-soap-messages-via-dime
+seq:
+  - id: records
+    type: record
+    repeat: eos
+types:
+  padding:
+    doc: |
+      padding to the next 4-byte boundary
+    seq:
+    - id: padding
```

Comment:
Please, indent collections content relative to their key

Reply:
I made the doc strings one-line.

Reply:
@Sijisu:

> I made the doc strings one-line.

That's nice, but I think @KOLANICH meant this:

```diff
     seq:
-    - id: padding
-      size: (4 - _io.pos) % 4
```

```diff
     seq:
+      - id: padding
+        size: (4 - _io.pos) % 4
```

Reply:
@Sijisu:

See also [KSY Style Guide > 1. General formatting](https://doc.kaitai.io/ksy_style_guide.html#general):


> Formatting of sequences MUST include an indent, i.e.:
>
> <table>
> <thead>
> <tr>
> <th>Good</th>
> <th>Bad</th>
> </tr>
> </thead>
> <tbody>
> <tr>
> <td>
> 
> ```yaml
> imports:
>   - foo
>   - bar
>
> ```
>
> </td>
> <td>
> 
> ```yaml
> imports:
> - foo
> - bar
>
> ```
>
> </td>
> </tr>
> </tbody>
> </table>

Reply:
Oh I see, I'm sorry :man_facepalming:

Reply:
Fixed